src/components/MainThreeJSComponent.tsx

```diff
@@ -1,12 +1,12 @@
 import React, { useEffect, useRef } from 'react';
 import * as THREE from 'three';
 
-import CHUN from '../assets/tiles/Regular/Chun.png';
-import MAN1 from '../assets/tiles/Regular/Man1.png';
-import HATSU from '../assets/tiles/Regular/Hatsu.png';
-import NAN from '../assets/tiles/Regular/Nan.png';
-import PEI from '../assets/tiles/Regular/Pei.png';
-import SOU1 from '../assets/tiles/Regular/Sou1.png';
+import REDDRAGON from '../assets/tiles/Regular/REDDRAGON.png';
```

Comment:
these changes are already in master, maybe ur master branch is not up to date? need to merge or rebase master into ur branch